tsdef/TrackPublication.d.ts

```diff
@@ -0,0 +1,15 @@
+import { EventEmitter } from 'events';
+import { Track } from './Track';
+
+export class TrackPublication extends EventEmitter {
+  isEnabled: boolean;
+  kind: Track.Kind;
```

Comment:
`Kind` is defined as `'audio' | 'video' | 'data'` within the `Track` namespace inside of `Track.d.ts`. Since both the class and namespace are exported, there is type merging. So we can import `Track` from `Track.d.ts` and then have access to `Track.Kind` type definition which is `string`.
![image](https://user-images.githubusercontent.com/43423318/96044445-48081480-0e25-11eb-8e0c-cdae425541ef.png)
![image](https://user-images.githubusercontent.com/43423318/96044467-50604f80-0e25-11eb-94d7-66417d2a6c25.png)